### DISCUSSION TITLE
Could you support Absorption Chess or is there already a way to implement it through ini?

### DISCUSSION THREAD

**Discussion by MidnightStuff** [fairy-stockfish/Fairy-Stockfish#808] [Ideas]
I can't seem to find any Absorption Chess variants for free to play against AI. It's hard to believe that I'm the only one who thinks it's a fun concept. I tried to get Gemini AI to create an Absorption config for me, lol, but it said that the engine needs to be modified to support the rules. Is that true? Is so, that's my suggestion. 

Thanks.

**Comment by ianfab** [comment]
Yes, absorption chess is currently not supported, since movement modifications are generally tricky to handle.

PS: It generally is useful to have a short description of or link to rules when first mentioning a variant to not require to google it first.
